pkg/consts/metric.go

```diff
@@ -55,6 +55,10 @@ const (
 	MetricIOReadSystem  = "io.read.system"
 	MetricIOWriteSystem = "io.write.system"
 	MetricIOBusySystem  = "io.busy.system"
+
+	MetricIOReadSystemStat  = "io.read.system.stat"
+	MetricIOWriteSystemStat = "io.write.system.stat"
+	MetricIOBusySystemStat  = "io.busy.system.stat"
```

Comment:
Ambiguous naming since 'stat' does not imply rate meaning. MetricIOXxxRateSystem and io.xxx.rate.system might be better.